### DISCUSSION TITLE
[PROPOSAL] Animal Tweaks

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#4866] [General]
Hello, below is a proposal put forth by the team to update some of the values for animal XP and drops. This is aimed at providing a simpler solution to increase player engagement with the animal mechanic.

### Changes to Animal XP:

Sheep XP will convert similarly to the Chicken's.
lvl 1 - 60
lvl 2 - 120
lvl 3 - 240
lvl 4 - 360
lvl 5 - 480
lvl 6 - 660
lvl 7 - 840
lvl 8 - 1020
lvl 9 - 1200
lvl 10 - 1440
lvl 11 - 1680
lvl 12 - 1920
lvl 13 - 2160
lvl 14 - 2400
lvl 15 - 2700

Cow XP will convert to Sheeps'.
lvl 1 - 120
lvl 2 - 240
lvl 3 - 480
lvl 4 - 720
lvl 5 - 960
lvl 6 - 1320
lvl 7 - 1680
lvl 8 - 2040
lvl 9 - 2400
lvl 10 - 2880
lvl 11 - 3360
lvl 12 - 3840
lvl 13 - 4320
lvl 14 - 4800
lvl 15 - 5400

### Changes to Animal yield:

Chicken.
level 1: 1 egg
Level 2: 1 egg 
Level 3: 1 egg 1 feather
Level 4: 1 egg 1 feather
Level 5: 1 egg 1 feather
Level 6: 2 egg
Level 7: 2 egg 1 feather
Level 8: 2 egg 1 feather
Level 9: 2 egg 2 feather
Level 10: 2 egg 2 feather 
Level 11: 3 egg
Level 12: 3 egg 1 feather
Level 13: 3 egg 2 feather
Level 14: 3 egg 2 feather
Level 15 3 egg 3 feather
Level 15+: repeat level 15 rewards

### Changes to bounties:

More bounties will be available at base AND per upgrade.
10 bounties available at base and +5 per upgrade. Meaning a fully upgrade hen house (without boosts) would host 20 chickens and have 20 bounties for the week.

_Edited some values below_
Hen House:
> 10 bounties at base:
> 6 tickets, 4 coins

> 15 at 1st upgrade:
> 6 tickets, 9 coins

> 20 at 2nd upgrade:
>  6 tickets, 2 gems, 12 coins

Barn:
> 10 bounties at base:
> 6 tickets, 4 coins
﻿
> 15 at 1st upgrade:
> 6 tickets, 9 coins
﻿
> 20 at 2nd upgrade:
>  6 tickets (3 cow, 3 sheep), 2 gems (1 cow, 1 sheep), 12 coins (6 cow, 6 sheep)

IMPORTANT TO NOTE: These changes will not go live until auction week is finished.. so to not impact the seasonal pvp race mid-way through it.

I'd personally still like to change more in regards to this mechanic, but these can be some quick wins the team can do in a short amount of time. Also keep in mind, Animal Skills will be coming too..

We can also change Medicine recipe to replace iron with honey or other radish instead of eggs.. this is open for discussion too

Please be civil and respectful, note that values may change

**Comment by vitt0** [comment]
The changes to the XP and more bounties sound good, but I don't like the idea of adding more bounties tickets it would mean that to compete in future season basically everybody is oblided to have the last upgrades if they want to get a shot for the top items, doing something like this would be similar to reduce/increase the quantity of chores we get according to the expansions we have which Imo would be a bad change. 

I would rather leave the extra ones only for coins and some with gems for the 2nd upgrade sound good too

**Reply by TheDcol** [reply]
> I would rather leave the extra ones only for coins and some with gems for the 2nd upgrade sound good too

Yup I initially did not want to change the tickets. I agree with what you say in regards to making the extra ones coin only and few gems in the last tier.

Otherwise, anyone who has not upgraded to last level of house/barn will automatically lose.

for the 10, 15, 20 bounty breakdown:

10:
6 tickets, 4 coins

15:
6 tickets, 9 coins

20:
6 tickets, 12 coins, 2 gems (or 10 coins/4 gems.. can be tweaked)

enticing players to turn animals in is key, rewarding them with good coinage/gems is one way to do so.

**Comment by brian-man** [comment]
I have trouble evaluating the change.  Can you include the "from" not just the "to"?

Like what's the delta in this proposal for a L5 Chicken?

**Comment by raulmonoanivia** [comment]
I like some of the changes, but more tickets per building level seems unfair.
Other than that, I still think that the changes aren't enough yet. We need an infinite coin bounty for each animal in order to allow us to manage their life cycles adequately.

**Reply by brian-man** [reply]
> I like some of the changes, but more tickets per building level seems unfair. Other than that, I still think that the changes aren't enough yet. We need an infinite coin bounty for each animal in order to allow us to manage their life cycles adequately.

I'm not clear to what degree tickets vs coins would be included in the new bounties.

**Comment by Xeko-92** [comment]
Since we're making changes, I don't like that you're getting "less" at level 11 than level 10. Especially for sheep where the Merino Wool is so much more valuable than the regular Wool.

Also, I think there should be more incentive to level up animals. There should be a choice between selling for bounties and leveling them. Even with this change it's always going to be better to sell for bounties I think. You're punished so hard with the sick animals as well as the increased costs. The rewards need to be noticeably better at higher levels. Or some other benefit.

**Reply by TheDcol** [reply]
> Even with this change it's always going to be better to sell for bounties I think.

This is one of the intended goal of animals, to turn them in via bounties before they get expensive (unless there's a ticket bounty requesting a high lvl animal)

Lvl 15 bounties would have to reward the player a lot via bounty, which could be done. Actually make it worth it.

**Reply by Xeko-92** [reply]
It should be a choice that you actually need to think through or else why even have it be a choice? Just automatically sell them once they hit a certain level or something.

Currently, and with this change, there's a right way of playing and a wrong way, the right way is to sell them in bounties.

**Reply by TheDcol** [reply]
Part of the choice is the sickness component, which as it grows older the chance increases and hence the risk. 

yes it is better to turn the animal into bounties if you can, generally speaking. Unless you want the higher yield or to lvl them up for higher bounties.

I dont sell my high level chickens, I keep them as long as I can and only turn them into bounties when they get sick. Although, this is with golden egg gameplay :D It matters what kinds of boosts/items you have and ultimately whether you will go down the animal skills path too.

**Reply by Xeko-92** [reply]
What would be the issue with really increasing the higher level rewards so that people are making decisions on whether to keep them or not (outside of people with free feeding)? This feels like an easy layup and how it should be.

Games are about players making choices. If you're effectively taking away our choice (not literally, but even you are saying it's the "goal" to have players turn them in via bounties), what gameplay choices are we really making?

**Reply by TheDcol** [reply]
> What would be the issue with really increasing the higher level rewards so that people are making decisions on whether to keep them or not (outside of people with free feeding)? This feels like an easy layup and how it should be.

Ye that's what I'm saying, increase lvl 15 bounties :)

> Lvl 15 bounties would have to reward the player a lot via bounty, which could be done. Actually make it worth it.

**Reply by brian-man** [reply]
> Part of the choice is the sickness component, which as it grows older the chance increases and hence the risk.

Please consider adding some form of UI in the game so I can tell which of my cows is 25 days old so I can remove it ASAP.

**Reply by TheDcol** [reply]
> > Part of the choice is the sickness component, which as it grows older the chance increases and hence the risk.
> 
> Please consider adding some form of UI in the game so I can tell which of my cows is 25 days old so I can remove it ASAP.

I'd love for the age to show ingame, my initial proposal was to switch xp for hunger and show age as more prelevant and tie more things to it (yield/sickness/infection etc.)

**Reply by Xeko-92** [reply]
> Ye that's what I'm saying, increase lvl 15 bounties :)

That's not what I mean, I mean increasing the higher level rewards from the drops, so that players need to make an actual decision whether it's better for them to sell the animal for the bounty and risk animals getting sick or keeping them and getting the noticeably better rewards despite the higher cost. It's currently basically not a choice.

**Reply by TheDcol** [reply]
I see, the animal yields.

Is something akin to what I had above with chickens to yuor liking? Granted the feathers atm have a lacklustre use, but the general trend is that the higher chicken level, the more yield

**Reply by Xeko-92** [reply]
Like I said in my original post, I don't like that you're actually getting less from level 11 animals than you are from level 9s.

I also don't think that the top is quite enough to outweigh the costs & risk of sickness & forgoing handing them in for bounties.

Especially as we get more boosts, the difference would narrow even more. With a cowbell necklace, you're getting 3 milk at level 1 vs 5 milk at level 11 (assuming the values are the same for cows as they are for chickens) for 4x the xp needed, plus like I said, higher chances of getting sick and not handing the animals in for bounties.

**Reply by TheDcol** [reply]
> I don't like that you're actually getting less from level 11 animals than you are from level 9s.

I missed that this was referring to sheep yield which are not in this discussion.

One thing to remember, it cannot just be a net positive outcome for all at base value. We cannot just give out resources at a low cost to everyone.. well we could, but then the resources would devalue

We could have a look at cow/sheep yield and tune it so it gives more per level similarly to how chickens are displayed above

**Reply by Xeko-92** [reply]
> I missed that this was referring to sheep yield which are not in this discussion.

I made the educated guess that the other numbers would be the same for the other animals as that's how it works now. There was no indication that that would be the case as it wasn't mentioned.
 
> One thing to remember, it cannot just be a net positive outcome for all at base value. We cannot just give out resources at a low cost to everyone.. well we could, but then the resources would devalue

I'm not sure what you mean by this. There wouldn't be any number you could come up with that would be a net positive (well no number that we're even close to). I'm not talking about increasing the numbers at all levels, I'm talking about increasing the numbers at high levels compared to low levels to make up for the cost & risk of the animals being higher levels.

**Comment by felgaceltica** [comment]
I don't agree with the more tickets per level, the rest is ok to me.

Animal sickenes deserve a look to. Think it's very expensive now

**Reply by TheDcol** [reply]
https://github.com/sunflower-land/sunflower-land/discussions/4866#discussioncomment-11467258

Commented here regarding the tickets :)

sickness will be good to handle via skills :)

**Comment by Thebyber** [comment]
I quite like the proposal but including more ticket bountys per building level seems unfair to players who cannot afford the cost of upgrading the building. 

Therefore I would like to add the concept of chicken meat, sheep meat and beef. to reduce the coin bounty... you can make a bounty where you deliver the animal and it gives you more or less meat depending on the level of the animal. 

Meat is a basic food and new recipes could be made.

An example could be the following:

Hen House:

10 bounties at base:
6 tickets, 3 coins, 1 meat

15 at 1st upgrade:
6 tickets, 7 coins, 2 meat

20 at 2nd upgrade:
6 tickets, 2 gems, 8 coins, 4 meat

I would also add more meat bountys and coins for players who have the chicken coop

Hen House+coop:

12 bounties at base:
6 tickets, 4 coins, 2 meat

17 at 1st upgrade:
6 tickets, 8 coins, 3 meat

22 at 2nd upgrade:
6 tickets, 2 gems, 9 coins, 5 meat

Barn:

10 bounties at base:
6 tickets, 3 coins, 1 meat
﻿
15 at 1st upgrade:
6 tickets, 7 coins, 2 meat
﻿
20 at 2nd upgrade:
6 tickets (3 cow, 3 sheep), 2 gems (1 cow, 1 sheep), 8 coins (4 cow, 4 sheep), 4 meat (2 cow, 2 sheep)

**Reply by TheDcol** [reply]
Sorry i edited it before you posted, I changed the values to keep tickets the same at every upgrade <3

**Reply by Thebyber** [reply]
I already edited my numbers, the idea is to implement the meat and not distribute so many coin bountys... the meat could be used in many recipes and could be NOT tradable so that it has a value like oil.

**Reply by TheDcol** [reply]
Meat is interesting and I would like it to be in the game eventually.. I am cocnerned at the amount fo recipes we have ingame atm though :/ 

I wonder if we should retire some recipes first... we could get data on what is not being cooked/touched

**Reply by seijii36214** [reply]
I think just a flat 2 bounty increase from coop is too small. And it should also scale with level. It's a 20 bounties/20 max animals without a capacity boost. It would be nice if it is at least a little higher with coop:
Level 1 - 13/15
Level 2 - 21/25
Level 3 - 29/35

Or going along with your +2 but an additional +2 every level:
Level 1 - 12/15
Level 2 - 19/25
Level 3 - 26/35

**Reply by Thebyber** [reply]
> La carne es interesante y me gustaría que estuviera en el juego algún día. Sin embargo, me sorprende la cantidad de recetas que tenemos en el juego actualmente :/
> 
> Me pregunto si deberíamos retirar algunas recetas primero... podríamos obtener datos sobre lo que no se cocina/toca.

I think that there are many recipes that have become obsolete over time. It doesn't make much sense that a sunflower cake gives you less than 1k experience compared to cheesecake that gives you more than 50k experience. fermented carrots give you 400 exp in a day while making shroom syrup It takes less than 1 minute and gives you more than 16k experience. These are a couple of examples, many recipes really should be renewed.

Maybe it's time to renew all the recipes from the kitchen stalls and make new recipes with the new season of seasons.

**Comment by heatedlime** [comment]
> Hello, below is a proposal put forth by the team to update some of the values for animal XP and drops. This is aimed at providing a simpler solution to increase player engagement with the animal mechanic.
> 
> ### Changes to Animal XP:
> Sheep XP will convert similarly to the Chicken's. lvl 1 - 60 lvl 2 - 120 lvl 3 - 240 lvl 4 - 360 lvl 5 - 480 lvl 6 - 660 lvl 7 - 840 lvl 8 - 1020 lvl 9 - 1200 lvl 10 - 1440 lvl 11 - 1680 lvl 12 - 1920 lvl 13 - 2160 lvl 14 - 2400 lvl 15 - 2700
> 
> Cow XP will convert to Sheeps'. lvl 1 - 120 lvl 2 - 240 lvl 3 - 480 lvl 4 - 720 lvl 5 - 960 lvl 6 - 1320 lvl 7 - 1680 lvl 8 - 2040 lvl 9 - 2400 lvl 10 - 2880 lvl 11 - 3360 lvl 12 - 3840 lvl 13 - 4320 lvl 14 - 4800 lvl 15 - 5400
> 
> ### Changes to Animal yield:
> Chicken. level 1: 1 egg Level 2: 1 egg Level 3: 1 egg 1 feather Level 4: 1 egg 1 feather Level 5: 1 egg 1 feather Level 6: 2 egg Level 7: 2 egg 1 feather Level 8: 2 egg 1 feather Level 9: 2 egg 2 feather Level 10: 2 egg 2 feather Level 11: 3 egg Level 12: 3 egg 1 feather Level 13: 3 egg 2 feather Level 14: 3 egg 2 feather Level 15 3 egg 3 feather Level 15+: repeat level 15 rewards
> 
> ### Changes to bounties:
> More bounties will be available at base AND per upgrade. 10 bounties available at base and +5 per upgrade. Meaning a fully upgrade hen house (without boosts) would host 20 chickens and have 20 bounties for the week.
> 
> _Edited some values below_ Hen House:
> 
> > 10 bounties at base:
> > 6 tickets, 4 coins
> 
> > 15 at 1st upgrade:
> > 6 tickets, 9 coins
> 
> > 20 at 2nd upgrade:
> > 6 tickets, 2 gems, 12 coins
> 
> Barn:
> 
> > 10 bounties at base:
> > 6 tickets, 4 coins
> > ﻿
> > 15 at 1st upgrade:
> > 6 tickets, 9 coins
> > ﻿
> > 20 at 2nd upgrade:
> > 6 tickets (3 cow, 3 sheep), 2 gems (1 cow, 1 sheep), 12 coins (6 cow, 6 sheep)
> 
> IMPORTANT TO NOTE: These changes will not go live until auction week is finished.. so to not impact the seasonal pvp race mid-way through it.
> 
> I'd personally still like to change more in regards to this mechanic, but these can be some quick wins the team can do in a short amount of time. Also keep in mind, Animal Skills will be coming too..
> 
> We can also change Medicine recipe to replace iron with honey or other radish instead of eggs.. this is open for discussion too
> 
> Please be civil and respectful, note that values may change

The nerf to chicken eggs don’t make sense, can you elaborate?

You guys doubled egg requirements for all recipes, and egg prices are higher than they were before, thus making egg recipes way more expensive. This nerf makes sense if you guys half the egg requirements for recipes.

**Reply by heatedlime** [reply]
Also, would like to see medicine cost decrease, I like the idea of changing it from “iron” to “honey”. Why would animals eat a mineral lol? Honey and lemons would maybe make sense

**Reply by ThamaHalgravs** [reply]
in the real world garlic it's pretty used but we don't have it. At least for now

**Comment by Dionis404** [comment]
Will the mechanics of feeding remain the same? Will I need to feed the animals before I level up?
It seems to me that feeding once a day would be a more convenient and logical option.
After feeding, you will have resources, and the animal will have experience. This will make high-level animals more valuable.

**Reply by TheDcol** [reply]
Yes feeding is the same. I would eventually love a 'feed all' button automatically feeding all the animals at once, lots of clicking now.. D:

**Reply by Dionis404** [reply]
a lot of clicks, and it's very unclear how much food is needed for all the animals

**Reply by Thebyber** [reply]
The mechanic of giving animals happiness makes them gain experience 2 times a day before having to give the animals a lot of food to level up, perhaps the most logical thing is to give them food only once and level up more slowly so that they level up more slowly. This happiness mechanic makes more sense.

The animals should give you resources proportionally according to the food they receive.

**Reply by Asterion1989** [reply]
One think that bothering me, is about to feed multiply times with the correct food until the animal was satisfied. Maybe we can use a similar system of adding oil in the kitchen. We add some crops on the feeder machine, and it feeds the animals automatically, and all you need to do is "pet" or "medicate" them. 
![image](https://github.com/user-attachments/assets/1afb91fa-9bef-4f80-b6fd-d72ab6db81e8)

**Reply by poteitouweb3** [reply]
> One think that bothering me, is about to feed multiply times with the correct food until the animal was satisfied. Maybe we can use a similar system of adding oil in the kitchen. We add some crops on the feeder machine, and it feeds the animals automatically, and all you need to do is "pet" or "medicate" them. ![image](https://private-user-images.githubusercontent.com/78221179/392808085-1afb91fa-9bef-4f80-b6fd-d72ab6db81e8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM0NDMzMjgsIm5iZiI6MTczMzQ0MzAyOCwicGF0aCI6Ii83ODIyMTE3OS8zOTI4MDgwODUtMWFmYjkxZmEtOWJlZi00ZjgwLWI2ZmQtZDcyYWI2ZGI4MWU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA1VDIzNTcwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI4MmE0ZGI3YmFkNjg2OTdiMDQ3NGMzYzE0YjhhZjIxYTM5YzMyNWU0ZjQ5YWQzOWE3NzZlMTU3M2E5M2U2YTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ytGMapfxS4rQqbwNvxEOJeH8ebx1dL3M8KvxanYVFCI)

agree 100%! would be so much better.

**Comment by Dionis404** [comment]
![image](https://github.com/user-attachments/assets/a1cdbfe9-f0c7-424c-9a53-b7a310851488)

For convenience you can make that when you create a feed you get not 1 but 10, it will help to get rid of non-integer numbers will be on 4.5 and 45
![image](https://github.com/user-attachments/assets/28832875-e575-4730-9c47-fdabe4e64213)

**Comment by Kanzuki-L** [comment]
After this changes, those who own the infernal bullwhip, they are turn become profitable for sheep or cow. This changes at least feel better, but for those who doesn't have infernal bullwhip, will be difficult.

But i don't find where's the hen become better after the update, hen currently only those gold egg players able to sustain. But the rest all are having negative. And you may see all egg related nft value are reduced by 50% even more. I think we should have a better balance on hen. I do hope the chicken coop can have a better update, since the extra 1 egg doesn't really feel big different like before, also the extra hen also doesn't look like will let those who don't have gold egg players feel advantage.

Also for mixed grain, this really a bad idea tbh, reason because to made 1 mixed grain, we required 3 different crops, and the cost are very high. Most people will choose omnifeed to replace mixed grain. I know gems are the income for dev team, but with this kind of design, really hope there's a better balance on it.

For medicine, i think they should be a better way to create it, why don't we use honey or others to made 1 medicine ? Since hen produce only limited egg, but spend 10 eggs and 1 iron, the cost really high, and the sickness chances also quite high.

For leveling, it should make it less click, and appear the exact number of feed to level up, now it's too difficult to know how many feeds for one level to spend.

This is only my personal pov, i just speech out how i feel.

**Comment by seijii36214** [comment]
Feels quite weird that sheep will have the same xp as chickens. Maybe at least 1.5x its required xp. Then cow. would have 2.5x that of chickens' xp.

**Comment by TheDcol** [comment]
An update to the proposal:

___

### **- INCREASED BOUNTIES**
Hen House:
> 10 bounties at base:
> 8 tickets, 2 coins

> 15 at 1st upgrade:
> 8 tickets, 7 coins

> 20 at 2nd upgrade:
>  8 tickets, 2 gems, 10 coins

Barn:
> 10 bounties at base:
> 8 tickets (4 cow, 4 sheep), 2 coins (1 cow, 1 sheep)
﻿
> 15 at 1st upgrade:
> 8 tickets (4 cow, 4 sheep), 7 coins (4 cows, 3 sheep)
﻿
> 20 at 2nd upgrade:
>  8 tickets (4 cow, 4 sheep), 2 gems (1 cow, 1 sheep), 10 coins (5 cow, 5 sheep)

### **- CHANGE TO MEDICINE RECIPE**
From 10 eggs 1 iron -> 5 Lemon 3 Honey

### **- CHANGE TO CHICKEN YIELD**
Chicken.
level 1: 1 egg
Level 2: 1 egg 
Level 3: 1 egg 1 feather
Level 4: 2 egg 1 feather
Level 5: 2 egg 1 feather
Level 6: 2 egg 1 feather
Level 7: 2 egg 1 feather
Level 8: 3 egg 1 feather
Level 9: 3 egg 2 feather
Level 10: 3 egg 2 feather 
Level 11: 3 egg 2 feather 
Level 12: 3 egg 2 feather
Level 13: 4 egg 2 feather
Level 14: 4 egg 2 feather
Level 15 5 egg 3 feather
Level 15+: repeat level 15 rewards 

### **- ANIMAL XP**
We may adjust animal XP in the future, although we'd like to see how the above changes will impact engagement. We are also wary of changing animal XP before releasing Skills Revamp which include animal skills.. some of these will decrease the feed/increase output along with future changes such as the weather mechanic being teased for next year.

Animal Skills currently ingame for core team going through testing. these may change and/or others will be added on/removed. Not final:
![image](https://github.com/user-attachments/assets/16c5d044-2202-4a0e-a59e-ae997f6c4acd)

*bountiful Bounties is only coins*

**Reply by Thebyber** [reply]
Aren't the tier three skills too abusive? They are for specializing in only one animal and that's a pain. For example, if you specialize in sheep... you will only be able to make half of the bounties since making the cow bounties would mean a very high cost of feeding... If you have the gold egg it wouldn't affect you...

Also, if you take the sheepwise diet and cow-smart nutrition in the end you would be feeding 100% more to the chickens and 25% more to the cows and sheep each...

**Reply by Xeko-92** [reply]
Amazing changes, good job. Everything looks great.

My only complaint would be for those tier 3 skills. Are those strictly for people with the golden egg/cow? Because okay the cow one might be okay because of how much they cost, but would anyone else take the other ones?

I would say the sheep one could be okay if you could ONLY use sheep, but you "need" sheep because of the ticket bounties. Is there going to be a change to the bounties where you have a choice to turn in either a cow or a sheep?

**Comment by RicardoKeita** [comment]
Making Honey (Propolis) as medicine makes so much more sense!

**Reply by 01101010110** [reply]
Yes, please!

**Comment by Rudolph-the-Bumpkin** [comment]
Instead of thinking of ways for people to exit the feature, what if we actually give better incentives to players reach and keep their animals at lvl 15?

Example: Critical drops that would be possible from a specific level and have increased chances at higher levels

Instead of reducing the amount of food required per level, so people can feed them everyday and feel realized, what if we had a feeding daily cap that is equal to 1 food? 

This way: 
1- players will feel realized (daily chore done)
2- players won't need to spend lots of resources everyday, meaning they would have a chance to be self stuitanable without every single NFT (would need them some NFTs though, just like any other tree)
3- Animal resource would be more scarce, fixing the price/ROI problem
4- Players would have to strategize over animal bounties, since it would take longer to reach level 15 (since it would take days to level up a cow from 3 to 4, for example)
5- Feeding would require 1 click 
6- Players who spent years of dedication to reach lvl 100 wouldn't feel bad because someone that joined months ago also did it buy buying cheap milks)
7- Tools would have increased utility (thanks for bringing that out @Alisa haha)

To me, the animal feature is in a different "tempo". The game is about dedication over time. However, we can get a lvl 15 cow in less than 2 weeks.
Slow down things a bit! Let us dedicate time to raise a full grown animal. Then we will see people feeling BAD 🥲  when selling a level 15 cow. Not relieved 😮‍💨 
If that was the case, I would also give better coin rewards to high level animals

**Comment by bromowiliad** [comment]
Golden Cow is game-breakingly strong and the only answer to the ongoing discussion on Discord is that it's only going to a handful of players. 
Many powerful alternative boosts have been mentioned: a GA-like boost, sickness-immunity, +1 all animal products.

To level one cow from 0 ->15 with the current system takes roughly 300 corn + 345 wheat + 434 barley = 1080 crops. Even if we calculate all of that just as corn, that's over 400 SFL in crops being handed out for free PER cow PER 2-weeks. That's over 10,000 advanced crops for free per week for a barn of cows. There is only enough time in one week to harvest corn about 800 times (assuming 60 plots), so most players cannot physically farm enough resources in one week to maximize a single cow; you are about to hand that much value out TIMES TWENTY to a handful of players for free. 

You are giving the power to dictate the price of milk, leather, corn, wheat, and barley to GC holders. They could literally list everything at .01 SFL and still make hundreds of SFL/week for free.  

The average player produces fewer eggs per week than pre-rework, but eggs need to get nerfed because of Golden Egg holders.
New uses for wood have been implemented every season to maintain the price of wood because of Foreman Beaver.

These items have influenced design decisions that have made the game less and less and less appealing to new players over and over again. I have played SFL for over two years. A GC holder can easily make more SFL from that item in one month than I have from my entire span of playing this game. 

Games want more players entering than leaving. Giving broken power to a small population and then changing the game around those players is an unsustainable model that makes it increasingly harder to get and retain new players. I've stayed with SFL because I love the concept and have faith the team would learn from its mistakes and keep growing.  Releasing Golden Cow would show me you have not learned. 

I strongly feel Golden Cow should be shelved for now at least until players are happy with the animal system. The consensus is the system is broken, and this single item threatens to break it even more and irreparably.
I predict after GC, farming cows will go from unprofitable to "even more unprofitable to everyone except GC holders." If I'm wrong, I will learn from it; if I'm write, I will very likely start liquidating my farm.
